Extensions/better_reblogs.js

```diff
@@ -277,6 +283,9 @@ XKit.extensions.better_reblogs = new Object({
 		if (this.preferences.color_quotes.value) {
 			this.run_cq();
 		}
+		if (this.preferences.decrease_padding.value === true) {
+			XKit.tools.add_css(".xkit-better-reblogs-old blockquote.reblog-quote {padding-left: 8px; margin-top: 6px;} .xkit-better-reblogs-old p.reblog-user {margin-bottom: 6px;}", 'better_reblogs_decrease_padding');
```

Comment:
should be just `"better_reblogs"` so that it gets removed automatically in `destroy()`

Reply:
also, i think we prefer to space out single-line CSS, i.e. `{ margin-bottom: 6px; }`

Reply:
ok thx